routers/routes/routes.go

```diff
@@ -523,6 +523,10 @@ func RegisterRoutes(m *macaron.Macaron) {
 				return
 			}
 		})
+
+		m.Group("/branches", func() {
+			m.Get("/_new/*", context.RepoRef(), repo.CreateBranch)
```

Comment:
@ethantkoenig it would be right way but semantic-ui dropdown does not have enough events exposed to do that :(